workflow/Snakefile

```diff
@@ -16,7 +16,7 @@ rule all:
         expand(dst_path("output/{seqid}/.delivery.done"), seqid=matching_map().seqid),
         dst_path("tables_delivery.done"),
         expand(dst_path("plots/{seqid}.png"), seqid=matching_map().seqid),
-
+        expand(ws_path("output/{seqid}/{seqid}_heterogenous_snp.tsv"), seqid=matching_map().seqid),
```

Comment:
adding this result here, makes the delivery and filter_heterogeneity rule asynchronous. So can happen that the delivery start before the heterogeneity result is ready and then it will not be synchronized in the destination folder.
To avoid this move `ws_path("output/{seqid}/{seqid}_heterogenous_snp.tsv")` as a second input of the sync_outputs_folder rule (and remove it from the rule all)